vyper/venom/__init__.py

```diff
@@ -55,8 +56,11 @@ def _run_passes(fn: IRFunction, optimize: OptimizationLevel) -> None:
     Mem2Var(ac, fn).run_pass()
     MakeSSA(ac, fn).run_pass()
     SCCP(ac, fn).run_pass()
+
     StoreElimination(ac, fn).run_pass()
     SimplifyCFGPass(ac, fn).run_pass()
+    LoadElimination(ac, fn).run_pass()
```

Comment:
it could be good to run `StoreElimmination` after this since it creates store chains

Reply:
Or just run it before it

Reply:
i checked, and it doesn't seem to change bytecode output (that maybe just because the optimization is not sophisticated enough yet)

Reply:
resolved in https://github.com/vyperlang/vyper/pull/4265/commits/cac7f8aa284c32f4190811f6fe186d81dfec963c